### PR TITLE
Updates to podcast content

### PR DESCRIPTION
**Pull request by Nevraeka**
On a side note - How is @kenchris never been added to authors? lol

### CODE REVIEW

File: authors.js [hunk 1]
```diff
@@ -11,6 +11,12 @@ module.exports = {
         github: "1240073",
         twitter: "onishiweb"
     },
+    alex_Sexton: {
+        name: "Alex Sexton",
```
Comment (samuelli) [inline authors.js]: Lower case

File: documents/podcasts/one-part-beer-three-parts-web-components.html.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: "One part beer, three parts web components. Add lemon to taste"
+authors: [trey_shugart, jem_young, ryan_anklam, brian_holt, stacy_london]
+podcast: "Front End Happy HOur"
```
Comment (samuelli) [inline documents/podcasts/one-part-beer-three-parts-web-components.html.md]: Hour

File: documents/podcasts/123-universal-web-components.html.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: "123: Universal Web Components"
+authors: [jordan_last, justin_ribeiro, leon_revill, danny_blue]
+podcast: "The Web Platform Podcast"
+date: 2017-02-01
+original_date: 2017-02-01
+link: "http://thewebplatformpodcast.com/123-universal-web-components"
+category: podcasts
+layout: single
```
Comment (samuelli) [inline documents/podcasts/123-universal-web-components.html.md]: FYI, layout line shouldn't be needed anymore, but it's fine to leave it.

File: authors.js (hunk 1 above)
Comment (kenchris) [reply authors.js]: The "S" :)

File: authors.js [hunk 4]
```diff
@@ -490,6 +532,18 @@ module.exports = {
...
     },
+    kito_mann: {
+        name: "Kito Mann",
+        bio: "Kito is a Principal Consultant at Virtua",
+        github: "3187538",
+        twitter: "kito99"
+    },
+    lars_knudsen: {
+        name: "Lars Knudsen",
+        bio: "Lars is CEO at IT Knows ApS",
```
Comment (larsgk) [inline authors.js]: "Founder of IT Knows, web platform and hardware connectivity evangelist"

File: authors.js (hunk 1 above)
Comment (Nevraeka) [reply authors.js]: nice catch. thx

**Comment by Nevraeka** [comment]
thank you @samuelli @larsgk @kenchris

**Comment by Nevraeka** [comment]
Can you merge this please @samuelli ? (I no longer have write access). Thank you :)

**Comment by Nevraeka** [comment]
Any updates on this? Are the changes ok? I can add the latest 2 ep of The Web Platform if you are waiting on this

**Comment by samuelli** [comment]
> Any updates on this? Are the changes ok? I can add the latest 2 ep of The Web Platform if you are waiting on this

Update is me getting back from vacation :).

You can send another PR for the latest eps.

**Pull request merged**